plugin.video.twitch/resources/language/Czech/strings.xml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
```

Comment:
Please change to strings.po for your next update, strings.xml is now deprecated.